zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -288,9 +306,9 @@ static String getValue(String key) {
      * @param key          The name of the configuration value.
      * @param defaultValue The default value to be used if configuration is missing.
      * @return The system property corresponding to the given key (e.g.
-     *         -Dkey=value). If that is blank, the environment variable
```

Comment:
I don't think these changes are needed, I think that the Javadoc was correct before.

Reply:
fixed indentation here